app/js/site.js

```diff
@@ -29,12 +29,14 @@ function getRandomIntInclusive(min, max) {
 function initalizeFirebase() {
 	
   // Initialize Firebase
```

Comment:
I agree on you.
Will change to common firebase server instead, when pull request to this master :)